conf/default.conf

```diff
@@ -5,4 +5,16 @@ server {
   root ./www;
   autoindex on;
   index index.html;
+
+
+  # Redirect /old to /new
+ location /old {
+    redirect 301 ./new;
+  }
+  location /new {
+    root            ./var/www/new;
+    allow_methods   GET;
+    autoindex       on;
+    index           new.html;
+  }
```

Comment:
Revert `default.conf` changes, keeping it as basic as possible. Add test config and `www/new/index.html`.